frontend/src/components/TextInput/TextInput.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './TextInput.css';
+import 'typeface-montserrat';
+
+function TextInput({
```

Comment:
Add NumberInput too (same as text, but only allows digits). EDIT: in its own file